.circleci/unittest/linux/scripts/run_test.sh

```diff
@@ -15,8 +15,15 @@ python -m torch.utils.collect_env
 export TORCHAUDIO_TEST_FAIL_IF_NO_EXTENSION=1
 export PATH="${PWD}/third_party/install/bin/:${PATH}"
 
+declare -a common_args=(
+    '--cov=torchaudio'
+    "--junitxml=${PWD}/test-results/junit.xml"
+    '--durations' '20'
+)
 if [ "${os}" == MacOSX ] ; then
-    pytest -q -n auto --dist=loadscope --cov=torchaudio --junitxml=test-results/junit.xml --durations 20 test
+    declare -a args=('-q' '-n' 'auto' '--dist=loadscope')
 else
-    pytest -v --cov=torchaudio --junitxml=test-results/junit.xml --durations 20 test
+    declare -a args=('-v')
 fi
+cd test
+pytest "${args[@]}" "${common_args[@]}" torchaudio_unittest
```

Comment:
could the following command work without moving files from `test/` to `test/torchaudio_unittest/`?
```
pytest "${args[@]}" "${common_args[@]}" test
```

Reply:
An alternative would be remove source `torchaudio` directory.
1. do `rm torchaudio` before running test - ugly
2. Move the source code to `src`. See [this](https://blog.ionelmc.ro/2014/05/25/python-packaging/#the-structure). - though this is clean and standard way, it involves the reorg of whole source code.

Reply:
Nice, thanks for pointing these out! I'm ok with the approach suggested here -- though neither pytorch nor the other libraries do it.

cc @cpuhrsch @fmassa @zhangguanheng66